main.cpp

```diff
@@ -8,12 +8,38 @@
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
 #include <glm/glm.hpp>
+#include <nlohmann/json.hpp>
 
+#include <fstream>
 #include <iostream>
 #include <string>
 
 using namespace pong;
 
+nlohmann::json OpenJsonFile(const std::string& filePath)
```

Comment:
You'll have to let me know what you think of this library. It is the big json library on the block but I've never used it myself